### PR TITLE
194_Update_Tag_Your_Load_Section

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses issue:
https://app.zenhub.com/workspaces/open-forest-58f8bbe105a35fad2d275a0c/issues/usdaforestservice/usfs-timber-permitting/194

This PR addresses all aspects of the 194 card, making content updates to the main info page for each forests, adding in the "tagging your load section" and changing the wood cost and minimum number of cords for the Hiawatha forest.

## This pull request is ready to merge when...
- [x] Feature branch starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [ ] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [ ] This code has been reviewed by someone other than the original author﻿## Summary

### CODE REVIEW

File: frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html [hunk 1]
```diff
@@ -8,7 +8,7 @@ <h1 id="skip-nav-target" tabindex="-1">
       <span>Buy a firewood permit</span>
     </h1>
     <div *ngIf="!showRules">
-      <h2 *ngIf="forest">You are purchasing a...</h2>
+      <h2 *ngIf="forest">You are purchasing a... {{forest.minCost}}</h2>
```
Comment (mgwalker) [inline frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html:11]: Should the min cost be here or is this leftover from testing/debugging?

**Review by mgwalker — CHANGES_REQUESTED** [review]
Code looks fine. I have one question about whether adding a `min_cost` field is the best approach or if it'd be better to compute the minimum cost based on the minimum number of cords and cost per cord. It seems to me that adding a static field whose value is computed from other static fields creates extra data management.

There's a tickle in the back of my brain about DBAs discouraging storing computed values and instead computing them as-needed. Maybe something about normalization?

This is just a question. If you think there's a good reason to prefer storing the computed value, I'll happily mark this as approved, but I wanted to make sure we considered it first. (I'll prioritize approving this PR whenever y'all respond so you won't have to wait around too long.)

File: frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html (hunk 1 above)
Comment (Dmac26) [reply frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html:11]: @mgwalker - To be completely honest. I 100% agree with you. I'd prefer that the value was computed. I'm sure it's just my general noobishness when it comes to Angular, but I haven't, up to this point been able to make it work. I keep getting a NaN value or nothing at all when I do the computation on the .ts file.

Comment (mgwalker) [reply frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html:11]: Gotcha. In the name of keeping moving, then, let's do it this way and maybe sometime in the future someone can change it over. 😄

**Review by mgwalker — APPROVED** [review]

File: frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html (hunk 1 above)
Comment (Dmac26) [reply frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html:11]: @mgwalker - that is leftover. I'll remove it. Gratsi!

**Comment by Dmac26** [comment]
@mgwalker - In regards to your question about adding an additional column with a static value, I completely agree with you. I think it's just my relative noobishness here, but I wasn't able to get a computed value to work. I kept getting NaN value or nothing would render at all. I was trying to come up with a fix and this worked. Once I had it implemented though I didn't really like it. do you happen to know some reasons why the .ts would like a simeple statement like so: `this.minCost = this.forest.woodCost * this.forest.minCords;`?

I was putting that in the constructor.

**Comment by mgwalker** [comment]
I've been away from Angular so long, and it has changed completely since then, so I'm probably not a lot of help... 😬  But what I see on StackOverflow suggests using a property getter instead:

```ts
export class BuyFirewoodPermitComponent implements OnInit {
...
  get minCost() {
    return this.forest.woodCost * this.forest.minCords;
  }
..
}
```

I'm not sure why that'd work but setting a value in the constructor wouldn't, though, unless `this.forest` gets set some time later, in which case its values aren't available for the constructor to use. 🤔  And now that I've typed that out, I guess it makes sense. I dunno. I should probably brush up on my Angular. 😄

**Review by mtlaney — APPROVED** [review]

**Comment by Dmac26** [comment]
@mgwalker - Thanks for the tip on that! Easy enough. I made the change and I like that solution a lot better. Mike approved for me, so I'm merging in now.

**Pull request merged**